Assets/LeapMotionModules/InteractionEngine/Scripts/InteractionManager.cs

```diff
@@ -42,6 +42,10 @@ public class InteractionManager : MonoBehaviour {
     protected string _ldatPath = "InteractionEngine/IE.ldat";
 
     [Header("Interaction Settings")]
+    [Tooltip("The default Interaction Material to use for Interaction Behaviours if none is specified, or for Interaction Behaviours created via scripting.")]
+    public InteractionMaterial defaultInteractionMaterial;
```

Comment:
Variable should be protected, and only have public getters/setters if needed, to follow along with how other variables are treated in the manager.